src/libs/antares/study/parameters.cpp

```diff
@@ -49,11 +51,10 @@ namespace Antares
 {
 namespace Data
 {
-enum
-{
-    //! Hard coded maximum number of MC years
-    maximumMCYears = 100000,
-};
+
+//! Hard coded maximum number of MC years
+const uint maximumMCYears = 100000;
+
```

Comment:
We should check the number of MC years as an input from the user and assume it's valid later.